compiler/rustc_const_eval/src/transform/check_consts/ops.rs

```diff
@@ -331,6 +332,10 @@ impl<'tcx> NonConstOp<'tcx> for FnCallNonConst<'tcx> {
             ccx.const_kind(),
         ));
 
+        if let ConstContext::Static(_) = ccx.const_kind() {
+            err.note("consider using the `once_cell` crate: https://crates.io/crates/once_cell");
```

Comment:
I wonder if we could make this a structured suggestion, that suggests wrapping the const in `once_cell::Lazy::new(|| ... )`

Reply:
Maybe not worth it, but OTOH the current note is kinda vague -- like what am I supposed to do w/ the `once_cell` crate exactly?

Reply:
hmm, tbh I think even mentioning the crate name is enough - it points people to read the documentation and from there they can figure out what they want (`sync::OnceCell` or `sync::Lazy`). For context, the reason I opened the issue was someone who had never heard of `once_cell` or `lazy_static`, so just knowing the crate existed would have been enough to get them unstuck.

That said, `Lazy::new` seems fine and pretty close to the original intent.